src/main/java/com/google/api/codegen/discovery/transformer/nodejs/NodeJSSampleTypeNameConverter.java

```diff
@@ -92,18 +91,12 @@ public TypeName getTypeNameForElementType(TypeInfo typeInfo) {
   @Override
   public TypedValue getZeroValue(TypeInfo typeInfo) {
     // Don't call getTypeName; we don't need to import these.
-    if (typeInfo.isMap()) {
-      return TypedValue.create(new TypeName("Object"), "{}");
-    }
     if (typeInfo.isArray()) {
       return TypedValue.create(new TypeName("Array"), "[]");
     }
     if (PRIMITIVE_ZERO_VALUE.containsKey(typeInfo.kind())) {
       return TypedValue.create(getTypeName(typeInfo), PRIMITIVE_ZERO_VALUE.get(typeInfo.kind()));
     }
-    if (typeInfo.isMessage()) {
```

Comment:
Discussed offline, these pathways are not tested by any doc we currently have, but at the risk of failing for customers in the future we'll include handling of maps and objects.